### PR TITLE
Copy bcsymbolmap files to build products folder when archiving

### PR DESCRIPTION
**Pull request by kishikawakatsumi**
`*.bcsymbolmap` files are necessary to submit an app with including debug symbols. If there are no those files, following error occurs unless uncheck "include app symbols ..."

untitled

```
The archive did not contain 
<DVTFilePath:0x7f91ea526720:'...xcarchive/BCSymbolMaps/....bcsymbolmap'> as expected.
```

This PR makes it include the `*.bcsymbolmap` files to the each framework bundles. Those files are moved to an app bundle when archiving.

See also:
[Missing BCSymbolMap for AppStore Submission - Xcode7B5](https://forums.developer.apple.com/thread/14729)

/cc @jpsim @mrackwitz

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -111,6 +111,14 @@ xcrealmswift() {
     xc "-project $PROJECT $@"
 }
 
+copy_bcsymbolmap() {
```
Comment (tgoyne) [inline build.sh]: Could just be `find "$1" -name '*.bcsymbolmap' -type f -exec cp {} "$2" \;`

Comment (kishikawakatsumi) [inline build.sh]: Cool, thanks.

File: scripts/strip-frameworks.sh [hunk 2]
```diff
@@ -60,3 +60,12 @@ for file in $(find . -type f -perm +111); do
     fi
   fi
 done
+
+if [ "$ACTION" = "install" ]; then
+  echo "Copy .bcsymbolmap files to .xcarchive"
+  mv Realm.framework/*.bcsymbolmap RealmSwift.framework/*.bcsymbolmap "${CONFIGURATION_BUILD_DIR}"
+else
+  # Delete *.bcsymbolmap files from framework bundle unless archiving
```
Comment (jpsim) [inline scripts/strip-frameworks.sh]: this says archiving, but above you're checking for `"$ACTION" = "install"`. Are those actually the same?

**Comment by jpsim** [comment]
Out of curiosity, how much larger does this make the binaries?

**Comment by kishikawakatsumi** [comment]
@jpsim The size of each symbol files is 350KB for Realm and 150KB for RealmSwift.
For iOS has two architecture(armv7, arm64) and for OS X has no bitcode.
So total size of raw files is around 1.4MB. The symbol files are plain text files. So well compressed.

Compressed total size is around 200KB.

File: scripts/strip-frameworks.sh (hunk 2 above)
Comment (kishikawakatsumi) [inline scripts/strip-frameworks.sh]: Yes, `ACTION` variable is to be `build`, `clean`, `install`, `installhdrs`, and `installsrc`. I confirmed the variable will be `install` when archiving.

https://developer.apple.com/library/mac/documentation/DeveloperTools/Reference/XcodeBuildSettingRef/1-Build_Setting_Reference/build_setting_ref.html#//apple_ref/doc/uid/TP40003931-CH3-SW45

File: scripts/strip-frameworks.sh [hunk 3]
```diff
@@ -60,3 +60,11 @@ for file in $(find . -type f -perm +111); do
     fi
   fi
 done
+
+if [ "$ACTION" = "install" ]; then
+  echo "Copy .bcsymbolmap files to .xcarchive"
+  find . -name '*.bcsymbolmap' -type f -exec mv {} "${CONFIGURATION_BUILD_DIR}" \;
```
Comment (jpsim) [inline scripts/strip-frameworks.sh]: @kishikawakatsumi I've generalized this so `strip-frameworks.sh` works for any framework, like the rest of this script. Could you please test this?

**Comment by jpsim** [comment]
@kishikawakatsumi feel free to merge this once you've confirmed that my latest change is correct.

**Comment by kishikawakatsumi** [comment]
@jpsim Thanks. I've confirmed.

**Pull request merged**